.eslintignore

```diff
@@ -16,4 +16,4 @@ coverage
 pages/**/*.md
 
 # We shouldn't lint statically generated Storybook files
-storybook-static
+storybook-static
```

Comment:
@AugustinMauroy can you simply remove the changes you've made in these files? (I have the feeling you're still not doing rebasing correctly) as both this file, prettierignore and the package-lock.json were updated.

As mentioned before (including by others) you can simply cherry-pick your commits. There are even unrelated (unverified) commits in this PR from completely unrelated things.

Note that this PR cannot be merged before you fix your commit history.